wsgi/odyssy/announcement/models.py

```diff
@@ -0,0 +1,20 @@
+from __future__ import unicode_literals
+
+from django.db import models
+
+
+# Create your models here.
+
+class Announcement(models.Model):
+    """ Model for Announcement
+    key         -- Primary Key
+    initDate    -- Initialization Date
+    finDate     -- Final Date
+    title       -- Title of the Announcement
+    description -- Description of the Announcement
+    """
+    key = models.AutoField(primary_key=True)
+    initDate = models.DateTimeField()
+    finDate = models.DateTimeField()
+    title = models.CharField(max_length=50)
+    description = models.CharField(max_length=500)
```

Comment:
Please check if `initDate` and `finDate' are relatively valid or not before saving the objects.

![screenshot 61](https://cloud.githubusercontent.com/assets/25580776/26335702/5d173688-3f8b-11e7-9caa-e98859d38ce7.jpg)

As ou can see `initDate` is in 2018 and `finDate` is 2017 and I can see this object in latest announcements on the homepage.

![screenshot 59](https://cloud.githubusercontent.com/assets/25580776/26335782/dd6ad1e6-3f8b-11e7-9f61-d3ba72c5f7c1.jpg)